docs/management/admin/response-actions.asciidoc

```diff
@@ -1,4 +1,111 @@
 [[response-actions]]
 = Endpoint response actions
 
-This page is a placeholder for future documentation.
+The response console allows you to perform response actions on an endpoint using a terminal-like interface. You can enter action commands and get near-instant feedback on them. Actions are also logged in the endpoint's <<actions-log,actions log>> for reference.
+
+[NOTE]
+=====
+Response actions and the response console UI are https://www.elastic.co/pricing[Enterprise subscription] features. 
+=====
+
+[role="screenshot"]
+image::images/response-console.png[Response console UI]
+
+Launch the response console from various contexts throughout {elastic-sec}:
+
+* *Endpoints* page -> *Actions* menu (*...*) -> *Respond*
+* Endpoint details flyout -> *Take action* -> *Respond*
+* Alert details flyout -> *Take action* -> *Respond*
+
+To perform an action on the endpoint, enter a <<response-action-commands,response action command>> in the input area at the bottom of the console, then press *Return*. Output from the action is displayed in the console.
+
+NOTE: Some response actions may take a few seconds to complete. Once you enter a command, you can immediately enter another command while the previous action is running. 
+
+Activity in the response console is persistent, so you can navigate away from the page and any pending actions you've submitted will continue to run. To confirm that an action completed, return to the response console to view the console output or check the <<actions-log,actions log>>.
+
+[[response-action-commands]]
+== Response action commands
+
+The following response action commands are available in the response console.
+
+=== `isolate`
+<<host-isolation-ov,Isolate the host>>, blocking communication with other hosts on the network.
+
+Example: `isolate --comment "Isolate host related to detection alerts"`
+
+=== `release`
+Release an isolated host, allowing it to communicate with the network again.
+
+Example: `release --comment "Release host, everything looks OK"`
+
+=== `status`
+Show information on the host's status, including: {agent} status and version, policy status for the {endpoint-cloud-sec} integration, and when the host was last active.
+ 
+=== `processes`
+Show a list of all processes running on the host. This action may take a minute or so to complete.
+
+TIP: Use this command to get current PID or entity ID values, which are required for other response actions such as `kill-process` and `suspend-process`.
```

Comment:
we could add another tip here to explain that Entity ID's are unique on the host while PID's can be reused by the OS.  Because of this, Entity ID may be a more reliable choice when killing a process.

Reply:
Revised!